src/api/app/controllers/webui/patchinfo_controller.rb

```diff
@@ -274,6 +223,67 @@ def new_tracker
     render json: { error: error, issues: issue_collection }
   end
 
+  private
+
+  def read_patchinfo
+    @binaries = []
+    @file.each(:binary) do |binaries|
+      @binaries << binaries.text
+    end
+    @binary = []
```

Comment:
Is `@binary` needed? It doesn't seem to be in use.

Reply:
It seems it is not needed. Removed. Thanks!

Reply:
@dmarcoux, as @eduardoj is just moving the code, I would say it is not even in the scope of the PR to refactor these methods :thinking:  Anyway, it is already removed :wink: